utils_nlp/models/transformers/named_entity_recognition.py

```diff
@@ -54,7 +56,7 @@ def get_inputs(batch, model_name, train_mode=True):
             dict: A dictionary object contains all needed information for training or testing.
         """
 
-        if model_name.split("-")[0] not in ["bert"]:
+        if model_name.split("-")[0] not in ["bert", "distilbert"]:
```

Comment:
Maybe we can add a list_supported_models methods to this class to keep it synced with the classifier.

Reply:
yes I was about to do something similar. I don't like how we hard-code these names.